docs/control-plane.md

```diff
@@ -82,12 +82,12 @@ For the most advanced deployments, users may follow the Canary Rollout Pattern.
 
 * Start with a KFService that has a "stable" or "golden" default definition.
 * Modify the KFService to set the new ModelSpec under the `canary`.
-* Set trafficPercent to some integer value (e.g. 10)
+* Set canaryTrafficPercent to some integer value (e.g. 10)
 * Observe the KFService until the new revision progresses
-    * `trafficPercent` is sent to `canary`
-    * `100 - trafficPercent` is sent to `default`
-* Continue incrementing trafficPercent in accordance with your desired canarying strategy
-* Once complete, change the key `canary` to `default` and delete the original `default` section and `trafficPercent`.
+    * `canaryTrafficPercent` is sent to `canary`
+    * `100 - canaryTrafficPercent` is sent to `default`
+* Continue incrementing canaryTrafficPercent in accordance with your desired canarying strategy
```

Comment:
Can we hyperlink a more explicit example here - maybe expand TF flower example in docs/samples/Tensorflow repo?

Reply:
I am working on this :), will add a link here.